docs/UserGuide/Ecosystem Integration/Spark IoTDB.md

```diff
@@ -158,3 +159,29 @@ public class Example {
 }
 ```
 
+## Write Data to IoTDB
+
+### User Guide
+``` scala
+// import narrow table
+val df = spark.createDataFrame(List(
+      (1L, "root.test.d0",1, 1L, 1.0F, 1.0D, true, "hello"),
+      (2L, "root.test.d0", 2, 2L, 2.0F, 2.0D, false, "world")))
+
+val dfWithColumn = df.withColumnRenamed("_1", "Time")
+  .withColumnRenamed("_2", "device_name")
+  .withColumnRenamed("_3", "int")
+  .withColumnRenamed("_4", "long")
+  .withColumnRenamed("_5", "float")
+  .withColumnRenamed("_6", "double")
+  .withColumnRenamed("_7", "boolean")
+  .withColumnRenamed("_8", "text")
+dfWithColumn
+    .write
+    .format("org.apache.iotdb.spark.db")
+    .option("url", "jdbc:iotdb://127.0.0.1:6667/")
+    .save
+```
+
+### Notes
+1. You can directly write data to IoTDB whatever the dataframe contains a wide table or a narrow table.
```

Comment:
Give an example of importing wide table?

Reply:
got this